### PR TITLE
Layout/home page

### PR DESCRIPTION
**Pull request by renatamarques97**
add [color palette](https://github.com/renatamarques97/suplier_market/wiki/Color-Palette) for forms

Screen Shot 2020-10-04 at 12 36 48
Screen Shot 2020-10-04 at 10 29 20
Screen Shot 2020-10-04 at 14 06 08

### CODE REVIEW

**Comment by renatamarques97** [comment]
[Template do site (formulários)](https://trello.com/c/1W0fqgWW/39-template-do-site-formul%C3%A1rios)

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,21 @@
...
+    height: FFaker::Random.rand(10..80),
+    width: FFaker::Random.rand(10..80),
+    length: FFaker::Random.rand(10..80),
+    price: FFaker::Random.rand(10.0..80.0),
+    weight: FFaker::Random.rand(1.0..8.0).round(2),
+    quantity: FFaker::Random.rand(20..80),
+    person_type: "Provider",
+    person: provider
+  )
+end
```
Comment (AleBL) [inline db/seeds.rb:25]: I think it would be better not to send these seeds

**Review by AleBL — APPROVED** [review]

**Review by Lucaas-Eduardo — APPROVED** [review]
Nice

File: db/seeds.rb (hunk 1 above)
Comment (renatamarques97) [reply db/seeds.rb:25]: Well, the layout needs to have some products to visualize.

**Pull request merged**